docs/docs-content/clusters/pcg/deploy-app-pcg.md

```diff
@@ -0,0 +1,641 @@
+   [Increase PCG Node Count](./manage-pcg/scale-pcg-nodes.md) guide for more information.
+
+   | **Parameter**       | **Value**                                                                    | **Environment-Specific** |
+   | ------------------- | ---------------------------------------------------------------------------- | ------------------------ |
+   | **Number of Nodes** | `1`                                                                          | No                       |
+   | **Placement Type**  | `DDNS`                                                                       | No                       |
+   | **Search domains**  | Comma-separated list of DNS search domains. For example, `spectrocloud.dev`. | Yes                      |
+
+6. **Cluster Settings**
+
+   Set the parameter **Patch OS on boot** to `N`, meaning the OS of the PCG hosts will not be patched on the first boot.
+
+   | **Parameter**        | **Value** | **Environment-Specific** |
+   | -------------------- | --------- | ------------------------ |
+   | **Patch OS on boot** | `N`       | No                       |
+
+7. **vSphere Machine Configuration**
+
+   Set the size of the PCG as small (**S**) as this PCG will not be used in production environments.
+
+   | **Parameter** | **Value**                                     | **Environment-Specific** |
+   | ------------- | --------------------------------------------- | ------------------------ |
+   | **S**         | `4 CPU, 4 GB of Memory, and 60 GB of Storage` | No                       |
+
+8. **Node Affinity Configuration Information**
+
+   Set **Node Affinity** to `N`, indicating no affinity between Palette pods and control plane nodes.
+
+   | **Parameter**     | **Value** | **Environment-Specific** |
+   | ----------------- | --------- | ------------------------ |
+   | **Node Affinity** | `N`       | No                       |
+
+After answering the prompts of the `pcg install` command, a new PCG configuration file is generated, and its location is
+displayed on the console.
+
+```text hideClipboard
+==== PCG config saved ==== Location: /home/ubuntu/.palette/pcg/pcg-20240306182821/pcg.yaml
+```
+
+Next, Palette CLI will create a local kind cluster that will be used to bootstrap the PCG cluster deployment in your
+VMware environment. Once installed, the PCG registers itself with Palette and creates a VMware cloud account with the
+same name as the PCG.
+
+The following recording demonstrates the `pcg install` command with the `--config-only` flag. When using this flag, a
+configuration file named **pcg.yaml** is created under the path **.palette/pcg**. You can then utilize this file to
+install a PCG with predefined values using the command `pcg install` with the `--config-file` flag. Refer to the
+[Palette CLI PCG Command](../../palette-cli/commands/pcg.md) page for further information about the command.
+
+<Video title="palette-pcg-config-only" src="/videos/palette-pcg.mp4"></Video>
```

Comment:
This video shows `sp-carolina` and `spectrocloud.dev`. Perhaps blur?

Reply:
The playback speed was increased and the recording now is under 1 min :)